portal-bridge/src/cli.rs

```diff
@@ -92,6 +93,13 @@ pub struct BridgeConfig {
         help = "Data provider for consensus layer data. (\"pandaops\" / local node url)"
     )]
     pub cl_provider: Provider,
+
+    #[arg(
+        long = "el-provider-request-limit",
+        help = "Limits the amount of EL provider requests request day_limit / 86400 seconds. Ex. if set to 86400 we will only send EL 1 request a second",
```

Comment:
Again, I think here it's just a bit simpler if we condense the variable/help msg to simply the concept of a "daily limit", this is typically the limit people are concerned about (eg. the infura daily limit) .. and if they need to set the limit to on a "per second" basis, they can calculate the appropriate daily limit themselves
```rust
    long = "provider-daily-request-limit",
    help = "Maximum number of requests sent to the provider in a day."
    default_value_t = PROVIDER_DAILY_REQUEST_LIMIT,
)]
pub provider_daily_request_limit: f64,
```